solr/core/src/java/org/apache/solr/cli/ZkMvTool.java

```diff
@@ -69,18 +67,8 @@ public String getName() {
   public void runImpl(CommandLine cli) throws Exception {
     SolrCLI.raiseLogLevelUnlessVerbose(cli);
     String zkHost = SolrCLI.getZkHost(cli);
-    if (zkHost == null) {
```

Comment:
we are losing this special case messaging, however I think I am totally fine with that ;-).  Since I want to get rid of standalone mode anyway.